src/app/features/tech-record/create/create-tech-record.component.html

```diff
@@ -1,58 +1,28 @@
-<form [formGroup]="vehicleForm">
+<form [formGroup]="form">
   <h1 class="govuk-heading-l title">New vehicle information</h1>
 
-  <app-text-input
-    #inputVin
-    formControlName="vin"
-    name="input-vin"
-    label="Vehicle Identification number (VIN)"
-    [width]="30"
-    (ngModelChange)="ngOnChanges()"
-  ></app-text-input>
-
-  <app-text-input
-    #inputVrmOrTrailerId
-    formControlName="vrmTrm"
-    name="input-vrm-or-trailer-id"
-    label="VRM / Trailer ID"
-    [width]="30"
-    (ngModelChange)="ngOnChanges()"
-    *ngIf="!generateID.value"
-  ></app-text-input>
-
-  <span class="govuk-list" *ngIf="!generateID.value">Or</span>
+  <app-text-input formControlName="vin" [width]="30" (ngModelChange)="ngOnChanges()"></app-text-input>
+
+  <app-text-input *ngIf="!generateID.value" formControlName="vrmTrm" [width]="30" (ngModelChange)="ngOnChanges()"></app-text-input>
+
+  <span *ngIf="!generateID.value" class="govuk-list">Or</span>
 
   <app-checkbox-group
     #generateID
     formControlName="generateID"
-    name="generate-c-or-z-num"
-    [options]="[{ value: true, label: 'Generate a C/Z number on submission of the new record' }]"
+    [options]="checkboxOptions"
     (ngModelChange)="toggleVrmInput(generateID)"
   ></app-checkbox-group>
 
-  <app-select
-    #selectVehicleStatus
-    formControlName="vehicleStatus"
-    label="Select vehicle status"
-    [options]="vehicleStatusOptions"
-    name="change-vehicle-status-select"
-    (ngModelChange)="ngOnChanges()"
-  ></app-select>
-
-  <app-select
-    #selectVehicleType
-    formControlName="vehicleType"
-    label="Select vehicle type"
-    [options]="vehicleTypeOptions"
-    name="change-vehicle-type-select"
-    (ngModelChange)="ngOnChanges()"
-  ></app-select>
-
-  <ng-container *ngIf="this.isVinUniqueCheckComplete && !this.vinUnique">
-    <p class="govuk-error-message">The Vin your about to submit is a duplicate, if you are happy to continue please press continue</p>
-  </ng-container>
+  <app-select formControlName="vehicleStatus" [options]="vehicleStatusOptions" (ngModelChange)="ngOnChanges()"></app-select>
```

Comment:
I _think_ the name property is responsible for the id of the underlying component, should keep them in if that's the case